README.md

```diff
@@ -109,10 +114,11 @@ Compile and Deploy PerfKit Explorer
 
          cd ~/projects/PerfKitExplorer
 
-2. Modify the `app.yaml` file so that the instance class is appropriate for your
-   needs, and the application name matches the project id you created in the
-   'Create the App Engine project' step, and the version string is set
-   appropriately. For example:
+2. Modify the `app.yaml` file so that the
+   [instance class](https://cloud.google.com/appengine/docs/adminconsole/performancesettings)
+   is appropriate for your needs, and the application name matches the
+   project id you created in the 'Create the App Engine project' step,
+   and the version string is set appropriately. For example:
 
          application: perfkit-explorer-demo
          version: beta
```

Comment:
There's already an example in the original version, it just got truncated in the diff:

         application: perfkit-explorer-demo
         version: beta
         instance_class: F2

This reminds me, would be good to explain what the minimum resource requirements are. (I tried a F1 for the sample data.)